plugin/al/translators/pxrUsdTranslators/tests/CMakeLists.txt

```diff
@@ -1,10 +1,59 @@
+set(TEST_NAME TestPxrUsdTranslators)
+
+set(path 
+    "${CMAKE_INSTALL_PREFIX}/lib"
+    "${CMAKE_INSTALL_PREFIX}/plugin/pxr/maya/lib"
+    "${CMAKE_INSTALL_PREFIX}/plugin/al/lib"
+    "$ENV{PATH}"
+)
+
+set(pythonPath 
+    "${CMAKE_INSTALL_PREFIX}/plugin/al/lib/python"
+    "${CMAKE_INSTALL_PREFIX}/plugin/pxr/lib/python"
+    "${CMAKE_CURRENT_SOURCE_DIR}"
+    "$ENV{PYTHONPATH}"
+)
+
+set(mayaPluginPath 
+    "${CMAKE_INSTALL_PREFIX}/plugin/al/plugin"
+    "${CMAKE_INSTALL_PREFIX}/plugin/pxr/maya/plugin"
+)
+
+if(IS_WINDOWS)
+    string(REPLACE ";" "\;" pythonPath "${pythonPath}")
```

Comment:
turn into a utility function - see note for plugin/al/mayautils/AL/maya/tests/mayaplugintest/CMakeLists.txt